components/tidb_query_common/Cargo.toml

```diff
@@ -17,8 +17,8 @@ prost-codec = [
 ]
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
Is it possible to keep only depending on thiserror?

Reply:
I think we need anyhow for dynamic errors. IMO, we shouldn't have any dynamic errors, but in the short term, I think it is easier to keep using them and depend on anyhow. Longer term I have some ideas to transition away.